dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -160,7 +160,7 @@ def hidden_stage_ids
 
     stage_ids = current_user ? current_user.get_hidden_stage_ids(params[:script_id]) : []
 
-    render json: stage_ids
+    render json: stage_ids.map(&:to_s)
```

Comment:
The unit tests seem to expect each entry in this list to be quoted, hence the `to_s`.